--- origsrc/db-4.7.25/dist/aclocal/tcl.m4	2005-06-28 15:45:25.000000000 -0500
+++ src/db-4.7.25/dist/aclocal/tcl.m4	2008-07-10 21:19:01.984375000 -0500
@@ -110,7 +110,7 @@
 	aix4.[[2-9]].*)
 		LIBTSO_LIBS="$LIBTSO_LIBS $TCL_LIB_SPEC $TCL_LIB_FLAG"
 		LIBTSO_LIBS="$LIBTSO_LIBS -L$TCL_EXEC_PREFIX/lib -ltcl$TCL_VERSION";;
-	aix*)
+	aix*|cygwin*)
 		LIBTSO_LIBS="$LIBTSO_LIBS $TCL_LIB_SPEC $TCL_LIB_FLAG";;
 	esac
 	AC_SUBST(TCL_BIN_DIR)
