--- origsrc/db-4.7.25/dist/aclocal_java/ac_jni_include_dirs.m4	2007-11-16 09:48:09.000000000 -0600
+++ src/db-4.7.25/dist/aclocal_java/ac_jni_include_dirs.m4	2008-07-10 23:05:15.812500000 -0500
@@ -75,7 +75,6 @@
 	case "$host_os" in
 	aix*)		_JNI_INC_SUBDIRS="aix";;
 	bsdi*)		_JNI_INC_SUBDIRS="bsdos";;
-	cygwin*)	_JNI_INC_SUBDIRS="win32";;
 	freebsd*)	_JNI_INC_SUBDIRS="freebsd";;
 	hp*)		_JNI_INC_SUBDIRS="hp-ux";;
 	linux*)		_JNI_INC_SUBDIRS="linux genunix";;
--- origsrc/db-4.7.25/dist/configure.ac	2008-05-05 22:03:36.000000000 -0500
+++ src/db-4.7.25/dist/configure.ac	2008-07-10 23:11:17.390625000 -0500
@@ -411,7 +411,7 @@
         # A classpath that includes . is needed to check for Java
 	# Since Cygwin uses Windows' javac, we need Windows path separators
 	case "$host_os" in 
-	cygwin*)	CLASSPATH=".;$CLASSPATH";;
+	mingw*)	CLASSPATH=".;$CLASSPATH";;
 	*)		CLASSPATH=".:$CLASSPATH";;
 	esac
 	export CLASSPATH
