--- origsrc/db-4.7.25/dist/aclocal/tcl.m4	2005-06-28 15:45:25.000000000 -0500
+++ src/db-4.7.25/dist/aclocal/tcl.m4	2008-07-10 21:19:01.984375000 -0500
@@ -50,7 +50,7 @@
 
 	    # First check to see if --with-tclconfig was specified.
 	    if test "${with_tclconfig}" != no; then
-		if test -f "${with_tclconfig}/tclConfig.sh" ; then
+		if test -e "${with_tclconfig}/tclConfig.sh" ; then
 		    ac_cv_c_tclconfig=`(cd ${with_tclconfig}; pwd)`
 		else
 		    AC_MSG_ERROR([${with_tclconfig} directory doesn't contain tclConfig.sh])
@@ -59,8 +59,8 @@
 
 	    # check in a few common install locations
 	    if test x"${ac_cv_c_tclconfig}" = x ; then
-		for i in `ls -d /usr/local/lib 2>/dev/null` ; do
-		    if test -f "$i/tclConfig.sh" ; then
+		for i in `ls -d /usr/lib /usr/local/lib 2>/dev/null` ; do
+		    if test -e "$i/tclConfig.sh" ; then
 			ac_cv_c_tclconfig=`(cd $i; pwd)`
 			break
 		    fi
@@ -80,7 +80,7 @@
 AC_DEFUN(SC_LOAD_TCLCONFIG, [
 	AC_MSG_CHECKING([for existence of $TCL_BIN_DIR/tclConfig.sh])
 
-	if test -f "$TCL_BIN_DIR/tclConfig.sh" ; then
+	if test -e "$TCL_BIN_DIR/tclConfig.sh" ; then
 		AC_MSG_RESULT([loading])
 		. $TCL_BIN_DIR/tclConfig.sh
 	else
@@ -110,7 +110,7 @@
 	aix4.[[2-9]].*)
 		LIBTSO_LIBS="$LIBTSO_LIBS $TCL_LIB_SPEC $TCL_LIB_FLAG"
 		LIBTSO_LIBS="$LIBTSO_LIBS -L$TCL_EXEC_PREFIX/lib -ltcl$TCL_VERSION";;
-	aix*)
+	aix*|cygwin*)
 		LIBTSO_LIBS="$LIBTSO_LIBS $TCL_LIB_SPEC $TCL_LIB_FLAG";;
 	esac
 	AC_SUBST(TCL_BIN_DIR)
--- origsrc/db-4.7.25/dist/aclocal_java/ac_jni_include_dirs.m4	2007-11-16 09:48:09.000000000 -0600
+++ src/db-4.7.25/dist/aclocal_java/ac_jni_include_dirs.m4	2008-07-10 23:05:15.812500000 -0500
@@ -75,7 +75,6 @@
 	case "$host_os" in
 	aix*)		_JNI_INC_SUBDIRS="aix";;
 	bsdi*)		_JNI_INC_SUBDIRS="bsdos";;
-	cygwin*)	_JNI_INC_SUBDIRS="win32";;
 	freebsd*)	_JNI_INC_SUBDIRS="freebsd";;
 	hp*)		_JNI_INC_SUBDIRS="hp-ux";;
 	linux*)		_JNI_INC_SUBDIRS="linux genunix";;
--- origsrc/db-4.7.25/dist/configure.ac	2008-05-05 22:03:36.000000000 -0500
+++ src/db-4.7.25/dist/configure.ac	2008-07-10 23:11:17.390625000 -0500
@@ -411,7 +411,7 @@
         # A classpath that includes . is needed to check for Java
 	# Since Cygwin uses Windows' javac, we need Windows path separators
 	case "$host_os" in 
-	cygwin*)	CLASSPATH=".;$CLASSPATH";;
+	mingw*)	CLASSPATH=".;$CLASSPATH";;
 	*)		CLASSPATH=".:$CLASSPATH";;
 	esac
 	export CLASSPATH
